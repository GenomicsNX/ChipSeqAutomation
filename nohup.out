Warning: Permanently added '218.199.68.150' (RSA) to the list of known hosts.
Permission denied, please try again.
