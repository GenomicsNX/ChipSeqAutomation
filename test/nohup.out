--2016-04-21 01:19:11--  ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF_000001405.33_GRCh38.p7/GCF_000001405.33_GRCh38.p7_genomic.gff.gz
           => ‘GCF_000001405.33_GRCh38.p7_genomic.gff.gz’
Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.10, 2607:f220:41e:250::13
Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.10|:21... connected.
Logging in as anonymous ... Logged in!
==> SYST ... 